src/components/User.vue

```diff
@@ -1,38 +1,141 @@
-                Following
-            </div>
-    </div>
+        <div class="info flex-col">
+          <div class="name text-base font-bold text-lm_dark dark:text-dm_white">{{ userInfo.name || userInfo.login }}</div>
+          <div class="handle text-xs pb-2 text-primary">@{{ userInfo.login }}</div>
+          <div class="joined text-xs">Joined {{ new Date(userInfo.created_at).toLocaleDateString() }}</div>
+          
+        </div>
+      </div>
 
+      <div class="text-xs pt-6 pb-8">
+        {{ userInfo.bio || 'This profile has no bio' }}
+      </div>
+    
+      <div class="stats flex flex-row justify-evenly rounded-2xl bg-lm_whitegrey dark:bg-dm_black py-5 px-4">
+        <div class="flex flex-col justify-center items-center gap-2">
+          <span class="text-xs">Repos</span>
+          <span class="text-base font-bold text-lm_dark dark:text-dm_white">{{ userInfo.public_repos }}</span>
+        </div>
+        <div class="flex flex-col justify-center items-center gap-2">
+          <span class="text-xs">Followers</span>
+          <span class="text-base font-bold text-lm_dark dark:text-dm_white">{{ userInfo.followers }}</span>
+        </div>
+        <div class="flex flex-col justify-center items-center gap-2">
+          <span class="text-xs">Following</span>
+          <span class="text-base font-bold text-lm_dark dark:text-dm_white">{{ userInfo.following }}</span>
+        </div>
+      </div>
+      
+        <div class="pt-6 flex flex-col gap-4 font-bold">
+          <div class="flex justify-start items-center gap-3">
+            <img class="bottom-icons" src="/assets/icon-location.svg" alt="location" />
+            <span v-if="userInfo.location" class="location text-xs">
+              {{ userInfo.location}}
+            </span>
+          </div>
+          
+          <div class="flex justify-start items-center gap-3">
+            <img class="bottom-icons" src="/assets/icon-website.svg" alt="location" />
+            <a v-if="userInfo.blog" :href=userInfo.blog class="blog text-xs hover:underline">
+              {{ userInfo.blog }}
+            </a>
+          </div>
+          
+          <div class="flex justify-start items-center gap-3">
+            <img class="bottom-icons" src="/assets/icon-twitter.svg" alt="location" />
+            <a v-if="userInfo.twitter_username" :href=userInfo.twitter_username class="twitter text-x hover:underline">
+              {{ userInfo.twitter_username }}
+            </a>
+          </div>
+          
+          <div class="flex justify-start items-center gap-3">
+            <img class="bottom-icons" src="/assets/icon-company.svg" alt="location" />
+            <span class="company text-sm">
+              {{ userInfo.company }}
+            </span>
+          </div>
+        </div>
+      
+    </div>
+    <div v-else class="error">{{ error }}</div>
+    
+    
 </template>
 
 <style>
 
+.bottom-icons{
+  color: white;
+  width:20px;
+}
+
+.card {
+  padding: 32px 24px 48px 24px;
```

Comment:
Not sure why you create custom classes, when you could use tailwind. Here as well: `pt-8`, `px-6`, `pb-12`